scripts/extensions/helloWorld/tsconfig.json

```diff
@@ -1,14 +1,28 @@
 {
     "compilerOptions": {
-        "sourceRoot": "src",
         "outDir": "dist",
-        "sourceMap": true,
         "module": "commonjs",
-        "target": "es5",
+        "target": "es6",
         "lib": [
             "dom",
             "es2017"
         ],
         "skipLibCheck": true,
+        "jsx": "react",
+
+        "strict": true,
+        "alwaysStrict": true,
+        "strictBindCallApply": true,
+        "strictFunctionTypes": true,
+        "strictPropertyInitialization": true,
+        "strictNullChecks": true,
+        "noImplicitAny": true,
+        "noImplicitReturns": true,
+        "noImplicitThis": true,
+        
+        "forceConsistentCasingInFileNames": true,
+        "noFallthroughCasesInSwitch": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,  
```

Comment:
I'm thinking if we should just make each extension use the root `tsconfig.json`

Reply:
Ok nevermind, I realized we should remove `['scripts']` and stuff like that from root config, not needed.

Reply:
The core wouldn't compile under strict flags. We could extend and overwrite core tsconfig, but that might not work well with extensions which will be in another repository so keeping it separate is easiest at the moment.